library/Freckle/App.hs

```diff
@@ -251,6 +252,12 @@ instance (Monad m, HasTracer app) => MonadTracer (AppT app m) where
 instance Applicative m => XRay.MonadTracer (AppT app m) where
   getVaultData = pure Nothing
 
+instance
+  (MonadUnliftIO m, HasSqlPool app, HasStatsClient app)
+  => MonadSqlTx (ReaderT SqlBackend (AppT app m)) (AppT app m)
+  where
+  runSqlTx = XRay.runDB
```

Comment:
`XRay` is legacy, and only exists because megarepo can't migrate from XRay to Datadog until November 1 (when we get an APM trial in place). You want you use `Freckle.App.Database.runDB` for this -- I don't _think_ that'll be any kind of issue (for example, I don't think need to duplicate anything like we do with `MonadTracer`).